rosidl_generator_c/rosidl_generator_c/__init__.py

```diff
@@ -138,11 +138,31 @@ def basic_value_to_c(type_, value):
     if 'boolean' == type_.typename:
         return 'true' if value else 'false'
 
-    if type_.typename in (*SIGNED_INTEGER_TYPES, *CHARACTER_TYPES, OCTET_TYPE):
+    if type_.typename in [
+        'short', 'unsigned short',
+        'char', 'wchar',
+        'double',
```

Comment:
This looks wrong. Please keep using the type constants possible: `CHARACTER_TYPES`, `OCTET_TYPE`.

Reply:
Hmm.. okay. I copied what was being done over in `rosidl_generator_cpp`:

https://github.com/ros2/rosidl/blob/d1edcaf35534e8860730370a62f1d6a024ee0f66/rosidl_generator_cpp/rosidl_generator_cpp/__init__.py#L176-L184

I'll update this file, but perhaps a follow-up is needed for rosidl_generator_cpp.

Reply:
e1c1b28